specifications/KeyValueStore/Util.tla

```diff
@@ -0,0 +1,49 @@
+-------------------------------- MODULE Util --------------------------------
+EXTENDS Sequences, FiniteSets, Naturals, TLC
+\* Simple utility functions
+intersects(a, b) == (Cardinality(a \cap b) > 0)
+max(s) == CHOOSE i \in s : (~\E j \in s : j > i)
+min(s) == CHOOSE i \in s : (~\E j \in s : j < i)
+
+\* Utilies from Practical TLA+
```

Comment:
The source of Practical TLA+ doesn't come with a [permissive license](https://github.com/Apress/practical-tla-plus/blob/master/LICENSE.txt).  The same or similar operators should exists in the CommunityModules.

Reply:
Util.tla came from/with ClientCentric.tla (which has a [permissive license](https://github.com/cwi-swat/tla-ci/blob/master/LICENSE))

It seems like only lines from 8-18 is implied to come from Practical TLA+. 
Would you be able to substitute with lines from community modules?

Reply:
https://github.com/tlaplus/CommunityModules/blob/master/modules/FiniteSetsExt.tla and https://github.com/tlaplus/CommunityModules/blob/master/modules/SequencesExt.tla appear to have similar operator definitions.

You should add a BSD license file in this folder since the examples repo is MIT licensed.

Reply:
Tried to fix things as much as I could, but I run into problems. So I am giving up.

Reply:
Ok, rewrote the three operators using examples from Community Modules, and added the license for the ClientCentric.tla

Should be ready to merge.

Reply:
Of course had to mention that Util now depends on Functions from Community modules. Arggh..